modules/auth/auth_form.go

```diff
@@ -24,11 +24,13 @@ type AuthenticationForm struct {
 	AttributeName                 string
 	AttributeSurname              string
 	AttributeMail                 string
+	AttributeSSHPublicKey         string
 	AttributesInBind              bool
 	Filter                        string
 	AdminFilter                   string
 	IsActive                      bool
 	IsSyncEnabled                 bool
+	IsLdapPublicSSHKeySyncEnabled bool
```

Comment:
I think there's a need for this option, consider the following case:
The administrator wants to synchronize LDAP users, but skip LDAP SSH Key synchronization and still have the AttributeSSHPublicKey value set.

It's not intuitive that setting the attribute automatically enables the key sync, as there's other options for enabling/disabling other login source features. If this option is not needed, I could argue that the "Enable user synchronization" checkbox and "This authentication is activated" is also not needed as those can test/check if the needed attributes for activation is set.

Given this input, do you still consider this option not needed?